mappings/net/minecraft/client/gui/WorldGenerationProgressTracker.mapping

```diff
@@ -6,9 +6,14 @@ CLASS net/minecraft/unmapped/C_qtaaqxsg net/minecraft/client/gui/WorldGeneration
 	FIELD f_upjrofby centerSize I
 	FIELD f_wdrmxewi size I
 	FIELD f_yrzklzbn running Z
+	METHOD m_bpcnlyan createWithTotalRadius (I)Lnet/minecraft/unmapped/C_qtaaqxsg;
+		ARG 0 spawnChunkRadius
+	METHOD m_inajsbju createWithRadius (I)Lnet/minecraft/unmapped/C_qtaaqxsg;
+		ARG 0 spawnChunkRadius
 	METHOD m_nmznsyyi getChunkStatus (II)Lnet/minecraft/unmapped/C_kogtzhzt;
 		ARG 1 x
 		ARG 2 z
 	METHOD m_npoofhgh getCenterSize ()I
 	METHOD m_obkavrre getSize ()I
+	METHOD m_ryrzyybt createWithoutSpawnRadius ()Lnet/minecraft/unmapped/C_qtaaqxsg;
```

Comment:
Not sure if these are descriptive enough, thoughts?

Reply:
these look good to me! i like that you delegated mentioning spawn chunks to the parameters